views/places.hbs

```diff
@@ -0,0 +1,18 @@
+<h1 class="page-title">{{localLang.placesTitle}}</h1>
+
+<ul>
+  {{#each output}}
+    <li class="place-wrap">
+      <img class="place-wrap__image" src="{{this.imageUrl}}" alt="Place Image">
+      <div class="place-wrap__info">
+        <span class="place-wrap__span">{{en.name}}</span>
+        <span class="place-wrap__span">{{en.description}}</span>
```

Comment:
So anything that's not `required` in the API schema [here](https://github.com/foundersandcoders/open-tourism-platform/blob/master/docs.md#create-place) we need to check whether or not it exists in the response